### DISCUSSION TITLE
New to AdvantEdge want guidence regarding docker images

### DISCUSSION THREAD

**Discussion by abmuslim** [InterDigitalInc/AdvantEDGE#368] [Q&A] [answered]
Hello,
I just started learning Advantedge and have some question arises in my mind. 
First thing first. I am using GUI. I made a simple topology where UE and edge nodes are made. My first Question is What should be the container image I should put in order to make my topology running and show that Ok! topology is working and there is communication going on. I am confused with container image tab option and Command and Argument part. 
Secondly when I done with Scenario then I am confuse with Execute tab like what exactly the manual, Auto replay, save etc represent Like how I run the topology there.
Thirdly how to monitor the results there are many options in monitor tab which one suits me?
And where is the programming part comes in the Advantedge? 
I am beginner already read the manual.
Thank you

**Comment by dilallkx** [comment, marked as answer]
Hi @abmuslim,

Thank you for using our platform and reaching out for more information!
I believe the answers to your questions can be found in the AdvantEDGE documentation:
- Scenario configuration (container image, command, arguments, etc.)
  - https://interdigitalinc.github.io/AdvantEDGE/docs/usage/gui/config-view/
- Execute tab (manual events, auto replay, save, etc.)
  - https://interdigitalinc.github.io/AdvantEDGE/docs/usage/gui/exec-view/
- Monitoring tab:
  - https://interdigitalinc.github.io/AdvantEDGE/docs/usage/gui/mon-view/

I would also recommend running the example scenario demo1 on your AdvantEDGE deployment as described in the platform usage documentation:
https://interdigitalinc.github.io/AdvantEDGE/docs/usage

After becoming familiar with the platform features, you can create scenarios that deploy and test your own edge application images.

I hope this helps you get started!
Regards!
